pages/design/rule-template.md

```diff
@@ -4,16 +4,15 @@ title: Auto-WCAG Rule Template
 
 The rule template contains a plain language description of the rule, some background information, and a list of all assumptions. The test procedure is defined by the selector, a number of steps and a description of the possible outcomes.
 
-Use the [empty test template](rule-template-empty.html) to create new auto-wcag rule. When creating a new rule, first read [rule design](rule-design.html).
+Use the [empty atomic rule template](atomic-template-empty.html) to create new auto-wcag rule. When creating a new rule, first read [rule design](rule-design.html).
 
-```markdown
+````markdown
```

Comment:
why ```` 4 backticks?

Reply:
Markdown magic. I needed to put triple backticks inside this block for the test cases. By giving the outer one an extra backtick, the markdown parser knows not to stop at my triple backticks for the test cases, but to continue until the expression with four backticks.

Reply:
Right, truly magical.